src/lib/plot/hint.js

```diff
@@ -23,11 +23,21 @@ import React from 'react';
 import PureRenderComponent from '../pure-render-component';
 import {getAttributeFunctor} from '../utils/scales-utils';
 
-const ORIENTATION_AUTO = 'auto';
-const ORIENTATION_TOPLEFT = 'topleft';
-const ORIENTATION_BOTTOMLEFT = 'bottomleft';
-const ORIENTATION_TOPRIGHT = 'topright';
-const ORIENTATION_BOTTOMRIGHT = 'bottomright';
+const ORIENTATION = {
+  AUTO: 'auto',
+  TOPLEFT: 'topleft',
+  BOTTOMLEFT: 'bottomleft',
+  TOPRIGHT: 'topright',
+  BOTTOMRIGHT: 'bottomright',
+  EDGETOP_LEFT: 'edgetopLeft',
```

Comment:
We should be clearer what these refer to. Perhaps a quick description or some ascii illustration?

Also, an alternative to improve readability: `EDGE_TOP__LEFT`.

On reading this without checking the code, I don't find it particularly clear what `EDGETOP_LEFT` does. It aligns to the top left, and the top edge?